### PR TITLE
First Companion Table Ingest

### PR DESCRIPTION
**Pull request by Ajb2307**
First ingest into Companions table

- [x] ingest reference not in Simple
- [x] ingest source not in Simple
- [x] import google sheets
- [x] write ingest function

### CODE REVIEW

File: REFERENCE_TABLES.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from astrodbkit2.astrodb import create_database,Database
+from simple.schema import *
+
+connection_string = 'sqlite:///SIMPLE.db.py' # connection string for a SQLite database named SIMPLE.db.py
+create_database(connection_string)
+db = Database(connection_string)
+
+db.load_database('data/')
```
Comment (kelle) [inline REFERENCE_TABLES.py:8]: Not sure where you got this from. Please use your `filip_15.py` script as an example and use the `load_simpledb` function.

File: SIMPLE.db-journal
Comment (kelle) [inline SIMPLE.db-journal:1]: This file should not be committed.

File: SIMPLE.db.py
Comment (kelle) [inline SIMPLE.db.py:1]: This file should not be committed.

File: REFERENCE_TABLES.py (hunk 1 above)
Comment (Ajb2307) [reply REFERENCE_TABLES.py:8]: I deleted that file, honestly no clue where it came from

**Review by Ajb2307 — COMMENTED** [review]
Add other names function I can only find in the  utils_deprecated file

**Comment by kelle** [comment]
Looking good!!

**Pull request closed without merging**